macosx.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+# This script contains common code for building PyNE on various Debian-derived systems
+#
+
+function check_repo() {
+
+    repo_name=$1
+
+    if [ -d $repo_name ] ; then
+        read -p "Delete the existing $repo_name directory and all contents? (y/n) " -n 1 -r
+        if [[ $REPLY =~ ^[Yy]$ ]] ; then
+            rm -rf $repo_name
+        fi
+    fi
+
+}
+
+function build_moab {
+
+    # Install MOAB
+    cd ${install_dir}
+    mkdir -p moab
+    cd moab
+    check_repo moab-repo
+    git clone --branch Version5.1.0 --single-branch https://bitbucket.org/fathomteam/moab moab-repo
+    cd moab-repo
+    mkdir -p build
+    cd build
+    cmake ../ -DENABLE_HDF5=ON -DHDF5_ROOT=${hdf5_libdir} \
```

Comment:
```suggestion
    cmake ../ -DENABLE_HDF5=ON \
```

this is not required, as hdf5_libdir is not defined, and brew installs it in a system known location

Reply:
`hdf5_libdir` is defined in the script that calls this, so you probably want to remove that line, too